hydrus/item_collection_helpers.py

```diff
@@ -85,7 +85,7 @@ def item_collection_get_response(path: str) -> Response:
             return error_response(error)
 
     # If endpoint and GET method is supported in the API and class is supported
-    if path in parsed_classes and f"{path}Collection" not in collections:
+    if path in parsed_classes:
```

Comment:
Does parsed classes not contain collections anymore? Why is this function being called for non-collection classes?

Reply:
There was a slight confusion regarding that code. 

The function was being called for non-collection classes to accommodate for classes such as 'Area' which have the same endpoint(from the POV of `flask`) like any collection of the form `/serverapi/classname`.